flexmeasures/data/services/account.py

```diff
@@ -7,3 +8,12 @@ def get_number_of_assets_in_account(account_id: int):
         GenericAsset.account_id == account_id
     ).count()
     return number_of_assets_in_account
+
+
+def get_account_roles(account_id: int):
```

Comment:
@GustaafL and @nhoening: I found this open conversation and it looks like an unresolved bug to me (which is now on main). If `account is None`, calling `account.account_roles` will crash.

I recommend adding this assert statement to the test `test_get_account_roles`:
```
assert get_account_roles(9999999) == []  # non-existing account id
```

By the way, when I run `pytest -k test_get_account_roles` the corresponding test also fails. This is because the test is missing the `setup_assets` fixture. It only passes when you run the entire test module, because a preceding test uses that fixture.